guides/logging.txt

```diff
@@ -0,0 +1,24 @@
+TOPIC
+    logging
+
+DESCRIPTION
+    Bolt logs messages both to the console and to log files. Messages can
+    either come from Bolt's internal 'outputter' class, which logs user-facing
+    messages like progress and results, or from the 'logger' class, which logs
```

Comment:
```suggestion
    either come from Bolt's outputter, which logs user-facing
    messages like progress and results, or from the logger, which logs
```

I think mentioning classes is more technical detail than a user needs to understand logging/output.